src/gg.regression.unity.bots/Runtime/Scripts/StateRecorder/ReplayDataPlaybackController.cs

```diff
@@ -2,8 +2,9 @@
 using System.Collections.Generic;
 using System.Linq;
 using StateRecorder;
-using TMPro;
-using UnityEditor.AnimatedValues;
+#if ENABLE_LEGACY_INPUT_MANAGER
+using RegressionGames.RGLegacyInputUtility;
+#endif
 using UnityEngine;
 using UnityEngine.EventSystems;
 using UnityEngine.InputSystem;
```

Comment:
Check line ~95 .. I think we need to take back out the code that errors / auto injects the new UI Input System